explore/src/main/scala/explore/attachments/package.scala

```diff
@@ -121,21 +121,23 @@ trait ObsAttachmentUtils:
             )
             .toastErrors
             .flatMap(id =>
-              (onSuccess(id) *>
-                obsAttachments
-                  .mod(
-                    _.updated(id,
-                              ObsAttachment(
-                                id,
-                                attType,
-                                name,
-                                None,
-                                false,
-                                f.size.toLong,
-                                Timestamp.unsafeFromInstantTruncated(Instant.now())
-                              )
-                    )
-                  )).toAsync
+              IO(Timestamp.unsafeFromInstantTruncated(Instant.now())).flatMap { now =>
+                (onSuccess(id) *>
+                  obsAttachments
+                    .mod(
+                      _.updated(id,
+                                ObsAttachment(
```

Comment:
Because we need to proactively update the local times. There is a `lastUpdated` time. It won't be exact with the server, but it's close enough because we only display the date.